geolocator/lib/geolocator.dart

```diff
@@ -261,6 +261,31 @@ class Geolocator {
         timeLimit: timeLimit,
       );
 
+  /// Fires every time NMEA-0183 sentences are received.
```

Comment:
I realize you probably copied this description from the `getPositionStream` but reading it again I see it can be a bit improved. Since the method returns a stream, nothing is really "fired" so a better description would be:

```suggestion
  /// Returns a stream emitting NMEA-0183 sentences when they are received.
```

Of course it would be great if you could also update the description for the `getPositionStream` in a similar way.